src/buildschemas.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Build all schemas by dereferencing the refs in the base schemas
```

Comment:
Where is this referenced? I havent managed to find a location?

Reply:
I think I included a separate file 'buildschemas.js' that does the job. Since you only need to run it once after changing the schema files, I didn't put it with the other files.